controllers/reviews/addReview/swagger.js

```diff
@@ -0,0 +1,111 @@
+export const addReview = {
+  components: {
+    schemas: {
+      Review: {
+        type: 'object',
+        required: ['id', 'owner', 'dish', 'rating', 'review'],
+        properties: {
+          id: {
+            type: 'string',
+            description: 'The auto-generated id of the review',
+          },
+          owner: {
+            type: 'string',
+            description: 'The id of the review owner',
+          },
+          dish: {
+            type: 'string',
+            description: 'The id of the reviewed dish',
+          },
+          rating: {
+            type: 'integer',
+            format: 'int32',
+            description: 'The rating given to the dish (integer value)',
+          },
+          review: {
+            type: 'string',
+            description: 'The review text',
```

Comment:
better to have max length for such text fields

Reply:
max 400